flake_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
`pytest --run-headless tests/form_autofill/test_edit_credit_card.py >> err_log.txt 2>&1` for logging would be helpful, esp if you have 10+ runs where the terminal would not keep all the logging history, also worth excluding incident tests? `pytest -m "not incident"` (Tracy found this one!)